.github/workflows/release-process_feature-freeze.yml

```diff
@@ -0,0 +1,46 @@
+name: Feature Freeze
+run-name: Feature freeze Cura ${{ inputs.cura_version }} by @${{ github.actor }}
+
+on:
+  workflow_call:
+    inputs:
+      cura_version:
+        description: 'Cura version number, e.g. 5.7.2'
+        required: true
+        type: string
+
+jobs:
+  parse-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version_major: ${{ steps.version_parser.outputs.major }}
+      version_minor: ${{ steps.version_parser.outputs.minor }}
+      version_patch: ${{ steps.version_parser.outputs.patch }}
+    steps:
+      - name: Parse version string
+        id: version_parser
+        uses: booxmedialtd/ws-action-parse-semver@v1.4.7
+        with:
+          input_string: ${{ inputs.cura_version }}
+
+  feature-freeze:
+    runs-on: ubuntu-latest
+    needs: [parse-version]
+    strategy:
+      matrix:
+        #repository: [Cura, Uranium, CuraEngine, cura-binary-data, fdm_materials]
+        repository: [Cura]
+    steps:
+      - name: Create branch and update conan package version
+        run: |
+          git clone https://${{ secrets.CURA_AUTORELEASE_PAT }}@github.com/Ultimaker/${{ matrix.repository }}.git
+          cd ${{ matrix.repository }}
+          BRANCH_NAME=${{ needs.parse-version.outputs.version_major }}.${{ needs.parse-version.outputs.version_minor }}
+          git checkout -b $BRANCH_NAME
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          PACKAGE_VERSION=${{ needs.parse-version.outputs.version_major }}.${{ needs.parse-version.outputs.version_minor }}.${{ needs.parse-version.outputs.version_patch }}
+          sed -i "s/version:.*/version: \"$PACKAGE_VERSION\"/g" conandata.yml
```

Comment:
it's yaml, just use python to read and modify it. way more robust and easier to maintain